ckanext/switzerland/templates/source/base.html

```diff
@@ -3,10 +3,10 @@
 {% block breadcrumb_content %}
   {% if harvest_source.organization %}
     {% snippet "snippets/organization_breadcrumb.html", organization=harvest_source.organization %}
-    <li>{{ h.nav_link(_('Harvest Sources'), named_route='{0}_search'.format(dataset_type)) }}</li>
-    <li{{ self.breadcrumb_content_root_selected() }}>{{ h.nav_link(harvest_source.title|truncate(10), named_route='{0}.read'.format(dataset_type), id=harvest_source.name) }}</li>
+    <li>{{ h.nav_link(_('Harvest Sources'), named_route='{0}_search'.format(c.dataset_type)) }}</li>
+    <li{{ self.breadcrumb_content_root_selected() }}>{{ h.nav_link(harvest_source.title|truncate(10), named_route='{0}.read'.format(c.dataset_type), id=harvest_source.name) }}</li>
   {% else %}
-    <li>{{ h.nav_link(_('Harvest Sources'), named_route='{0}_search'.format(dataset_type)) }}</li>
-    <li{{ self.breadcrumb_content_root_selected() }}>{{ h.nav_link(harvest_source.title|truncate(30), named_route='{0}.read'.format(dataset_type), id=harvest_source.name) }}</li>
+    <li>{{ h.nav_link(_('Harvest Sources'), named_route='{0}_search'.format(c.dataset_type)) }}</li>
+    <li{{ self.breadcrumb_content_root_selected() }}>{{ h.nav_link(harvest_source.title|truncate(30), named_route='{0}.read'.format(c.dataset_type), id=harvest_source.name) }}</li>
```

Comment:
This section is copied from ckanext-harvest, which has to maintain backwards compatibility with older versions of CKAN. I don't like including the `c` object here, but we get errors without it if we go to the harvest source admin page.